src/solvers/nonlinear_solid_solver.cpp

```diff
@@ -173,13 +173,13 @@ void NonlinearSolidSolver::completeSetup()
 
   // Set the MFEM abstract operators for use with the internal MFEM solvers
   if (timestepper_ == serac::TimestepMethod::QuasiStatic) {
-    solver_.solver().iterative_mode = true;
-    nonlinear_oper_                 = std::make_unique<NonlinearSolidQuasiStaticOperator>(std::move(H_form));
+    solver_.nonlinearSolver().iterative_mode = true;
+    nonlinear_oper_                          = std::make_unique<NonlinearSolidQuasiStaticOperator>(std::move(H_form));
```

Comment:
These two line giant spaces are making me twitch. I'll see if there's a way to get clang-format to suppress this.